source/hotkey.cpp

```diff
@@ -155,15 +155,14 @@ HotkeyCriterion *AddHotkeyIfExpr()
 
 HotkeyCriterion *FindHotkeyIfExpr(LPTSTR aExpr)
 {
-	for (HotkeyCriterion *cp = g_FirstHotExpr; cp; cp = cp->NextExpr)
-		if (cp->Type != HOT_IF_CALLBACK // i.e. HOT_IF_EXPR or an expression optimised to be any other type.
-			&& !_tcscmp(aExpr, cp->ExprLine->mArg[0].text)) // Case-sensitive since the expression might be.
+	for (HotkeyCriterion* cp = g_FirstHotExpr; cp; cp = cp->NextExpr)
+		if (cp->OriginalExpr && !_tcscmp(aExpr, cp->OriginalExpr)) // Case-sensitive since the expression might be.
 			return cp;
 	return NULL;
 }
 
 
-void Script::PreparseHotkeyIfExpr(Line *aLine)
+void Script::PreparseHotkeyIfExpr(Line* aLine)
```

Comment:
@Lexikos.
Visual studio does these changes on its own sometimes, it is not intended by me. I think the most common style is `type *var` vs `type* var`. I certainly miss and/or forget sometimes.